docs/subscription-plans.rst

```diff
@@ -115,141 +111,32 @@ case your preferred region is not currently available.
-directly via credit card, you do not have to concern yourself with DTUs. For a
-more detailed explanation of DTUs, see the section below.
-
-
-.. _subscription-plans-dtus:
-
-Explaining DTUs for scaling and billing
-=======================================
-
-What are DTUs and how do they work? As mentioned above, to make finding the
-right combination of hardware capacity more tractable and accessible, CrateDB
-Cloud's SaaS offers use DTUs. These DTUs have essentially two purposes: they
-allow the user to choose the right combination of plan and scale to find the
-capacity they need, and they provide clarity for the purposes of pricing. In
-order to keep things simple, scaling in each plan is currently set up so that
-one scale unit = one node = one DTU, and billing is set up so that Crate.io
-bills only for DTUs/hour actually used.
-
-Let's break this down further to clarify what each of these statements mean.
-
-As seen above, CrateDB Cloud's SaaS offer is divided into five plans. Each
-plan has a starting number of nodes (usually 3), that can be scaled to a larger
-number (between 4 and 15). Because the hardware capacity in each plan and tier
-is different, a node of the **GP** plan at the **Basic** tier, for example, is
-of a different size (in terms of storage, memory, and computation) than a node
-in the **I/O Optimized** plan at tier Premium. But the billing for these varied
-hardware compositions and scales is made easy by the fact that for any given
-plan and tier, one node corresponds to one DTU. Since for SaaS Marketplace
-users we bill in terms of DTUs/hour, all the different dimensions of horizontal
-and vertical hardware scaling are resolved into a single measure.
-
-An overview showing the range in terms of capacity of each SaaS plan and the
-price per DTU/hour can be found on the `Azure offer page`_ and the `AWS
-subscription page`_, respectively. All details about each hardware dimension,
-plan, tier, and node, can also be found on the deployment screen in the CrateDB
-Cloud console when deploying your cluster.
-
-To summarize:
-
-The DTU approach to scaling means that although the offered plans differ
-considerably in capacity per plan, tier, and number of nodes, the DTU system
-allows you to easily compare these different magnitudes for a single usage
-price on the SaaS Marketplaces.
-
-The precise calculations of hardware capacity, actual usage of that hardware,
-and a corresponding cost are all handled by Crate.io. The user only needs to
-choose a plan, a tier within that plan, and the number of nodes. You will then
-know the price in DTU/hour that corresponds to your choice.
+Marketplace offer
+=================
 
+If you have an existing Azure/AWS marketplace account and want to subscribe to
+CrateDB Cloud using that, you can. The principles are the same as with credit
+card subscription, which allows maximum flexibility regarding deployment and
+scaling up/down:
+
+- User is billed based on their consumption
```

Comment:
Technically the user is not billed, the organization is. But anyway do you think this reads better?
```suggestion
- Usage is billed based on consumption
```